src/types/publication.types.ts

```diff
@@ -139,18 +143,28 @@ const emptyPublicationEntityDescription: PublicationEntityDescription = {
   specialization: '',
   textBook: false,
   reference: {
+    type: 'Reference', //TODO: remove this when backend has fixed Reference
     doi: '',
     publicationInstance: {
+      type: 'PublicationInstance', //TODO: remove this when backend has fixed Reference
       volume: '',
       issue: '',
       articleNumber: '',
       peerReviewed: false,
       pages: {
+        type: 'Pages', //TODO: remove this when backend has fixed Reference
         begin: '',
         end: '',
       },
     },
-    publicationContext: null,
+    publicationContext: {
```

Comment:
hvorfor endre tilbake fra null til tomme objekter når man ikke har valgt publisher?

Reply:
pga vi må ha med Type for å lagre nå. backend jobber med det. det er bare midlertidig